### DISCUSSION TITLE
Visual Studio 2022 crashing?

### DISCUSSION THREAD

**Discussion by bernatagulloesbrina** [community/community#32686] [Copilot Conversations]
Hello, 
I'm having crashes with Visual Studio 2022 using copilot -- am I the only one? 
![image](https://user-images.githubusercontent.com/40455069/189548162-5f713310-2ebf-44f1-ade4-636c21562e26.png)

this didn't happen before copilot, but to be honest I've been using Visual Studio 2022 only for a short time so far, so not sure who the culprit is.

**Comment by Jonathan-Greve** [comment]
I think this is related to Github Copilot. I had the same error many times probably every 5-10 min and had to restart VS every time due to freezing.
Disabling Github Copilot fixed it.

When I have copilot enabled it works for a few minutes / few suggestions before it stops giving suggestions. Ctrl-Alt-Enter opens the Copilot window and it will show 'Loading Solutions' indefinitely. Only a restart can make it work again.

I would also occasionally get an error in Github Copilot output. Something about MoveNext().

Works perfectly fine in the VS code and Pycharm.
